inbox/mailsync/frontend.py

```diff
@@ -58,7 +58,7 @@ def profile():
         @app.route("/load")
         def load():
             if self.tracer is None:
-                return "Load tracing disabled\n", 404
+                return "Load tracing disabled\n"
```

Comment:
k8s uses this for health check, the tracer is greenlet specific and disabled on threads, if we do not return 200 here the pod would never become healthy